configs/prod/roles/patch.json

```
@@ -2,11 +2,11 @@
   "roles": [
     {
       "name": "Patch administrator",
-      "description": "Perform any available operation against any Patch resource.",
+      "description": "Perform any available operation on any Patch resource.",
       "system": true,
       "platform_default": false,
       "admin_default": true,
-      "version": 9,
+      "version":10,
```

not sure 100%, the missing space should be not a problem, but I'd rather change it
```suggestion
      "version": 10,
```